client/src/components/Tag/Tag.js

```diff
@@ -59,7 +59,7 @@ const Tag = ({
     <Button
       {...props}
       role="button"
-      className={classnames('tag', 'btn-sm', { tag__deletable: deletable })}
+      className={classnames('tag-component', 'btn-sm', { 'tag-component__deletable': deletable })}
```

Comment:
Deletable is a modifier in this case so technically it should be `tag-component--deletable`.